### ISSUE TITLE
Missing Field in Redis notification

### ISSUE THREAD

**Issue by sapiderman** [hyperjumptech/monika#962]
**Describe the bug**
1. On notification, the message and alerts are missing some useful info in the URL field.
2. It becomes unclear which probe caused the notification

**To Reproduce**
Config to reproduce the behavior:

```yaml


notifications:
  - id: id-desktop
    type: desktop
  - id: id-smtp
    type: smtp
    data:
      hostname: smtp.mysserver.com
      port: 2525
      username: usser
      password: password
      recipients: ['me@myserver.com']

  - id: id-teams
    type: teams
    data:
      url: https://mycompany.webhook.office.com/webhookb2/12345-1234-1234

probes:
  - id: id-redis
    name: redis-monitor
    description: testing redis
    interval: 10
    redis: 
      - host: 172.17.0.1
        port: 6380
    incidentThreshold: 3
    recoveryThreshold: 3
    alerts:
      - assertion: response.status != 200
        message: redis disconnected!

```

**Expected behavior**
1. Redis URI or probe-id should be displayed
2. No empty field in the notification messages

**Screenshots**
Teams Notification
![image](https://user-images.githubusercontent.com/964106/207574496-54bce8aa-d56c-4dde-b3c7-307a52617f3a.png)

SMTP Notification
![image](https://user-images.githubusercontent.com/964106/207575212-ad7412c5-e1c4-4318-a363-41371d6d9150.png)

**Desktop (please complete the following information):**
 - OS: PopOS! 22.04 LTS
 - Version: 1.14.1

**Comment by sapiderman** [comment]
So this turned out to be a simple problem with a pain fix.

Root cause is that some of the probes have `url`, some has `host` some has both. This info need to be extracted (and parsed on some).
Another solution maybe is not showing the url in the notification but just the `probe.id`

**Comment by guillaumemeyer** [comment]
`probe.id` would be fine, as long as we have a way to identify the source from the notification message.

**Comment by sapiderman** [comment]
Issue should be fixed after #1080... closing this issue.
